src/manifolds/GeneralUnitaryMatrices.jl

```diff
@@ -487,17 +487,47 @@ function get_vector_orthonormal!(
 end
 
 @doc raw"""
-    injectivity_radius(G::Orthogonal)
-    injectivity_radius(M::Rotations)
+    injectivity_radius(G::GeneraliUnitaryMatrices)
 
-Return the injectivity radius on the [`Rotations`](@ref) and the [`OrthogonalMatrices`](@ref) `M`,
-which is globally
+Return the injectivity radius for general unitary matrix manifolds, which is[^1]
 
 ````math
-    \operatorname{inj}_{\mathrm{O}(n)}(p) = \operatorname{inj}_{\mathrm{SO}(n)}(p) = π\sqrt{2}.
+    \operatorname{inj}_{\mathrm{U}(n)}(p) = π.
 ````
 """
+injectivity_radius(::GeneralUnitaryMatrices) = π
+
+@doc raw"""
+    injectivity_radius(G::GeneralUnitaryMatrices{n,ℂ,DeterminantOneMatrices})
+
+Return the injectivity radius for general complex unitary matrix manifolds, where the determinant is $+1$,
+which is[^1]
+
+```math
+    \operatorname{inj}_{\mathrm{SU}(n)}(p) = π \sqrt{2}.
```

Comment:
Given that `p` is not an argument to the function, it makes sense to remove it.
```suggestion
    \operatorname{inj}_{\mathrm{SU}(n)} = π \sqrt{2}.
```

Reply:
Well spotted :)